test_elasticsearch/test_async/test_server/test_helpers.py

```diff
@@ -332,6 +332,41 @@ async def test_error_is_raised(self, async_client):
         with pytest.raises(helpers.BulkIndexError):
             await helpers.async_bulk(async_client, [{"a": 42}, {"a": "c"}], index="i")
 
+    def test_ignore_error_if_raised(self, async_client):
+        # ignore the status code 400 in tuple
+        helpers.async_bulk(
```

Comment:
All of the calls to `async_bulk` need to be `await`-ed and run within an `async def test_...`

Reply:
Thanks for the great review.